### ISSUE TITLE
Passphrase text is cut in the cipher password dialog

### ISSUE THREAD

**Issue by revolter** [sqlitebrowser/sqlitebrowser#1424]
### Details for the issue

![screen shot 2018-06-09 at 16 36 48](https://user-images.githubusercontent.com/5748627/41192223-67484ac0-6c03-11e8-938d-93d66d13736b.png)

![screen shot 2018-06-09 at 16 36 53](https://user-images.githubusercontent.com/5748627/41192224-68eb6a7e-6c03-11e8-98cb-51dc71e2e6c7.png)

#### I'm opening this issue because:

- [ ] DB4S is crashing
- [x] DB4S has a bug
- [ ] DB4S needs a feature
- [ ] DB4S has another problem

#### I'm using DB4S on:

- [ ] Windows: ( _version:_ ___ )
- [ ] Linux: ( _distro:_ ___ )
- [x] Mac OS: ( _version:_ 10.13.4 (17E199) )
- [ ] Other: ___

#### I'm using DB4S version:

- [ ] 3.10.1
- [ ] 3.10.0
- [ ] 3.9.1
- [x] Other: 3.10.99 (Jun  8 2018)

#### I have also:

- [ ] Tried out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [ ] Searched for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by mgrojo** [comment]
Not happening for me under Ubuntu 16.04 (Qt5.5.1). It is even adapted to the long Spanish translation:

![imagen](https://user-images.githubusercontent.com/13812910/41193855-d13a97d0-6c12-11e8-8610-828be52a2626.png)

**Comment by revolter** [comment]
Maybe it's only on macOS

**Comment by justinclift** [comment]
Yeah, just make the widget a bit wider.  Should just be a value in some .ui file. :smile:

**Comment by revolter** [comment]
It's not that easy sadly.

Only the `MinimumExpanding` and `Expanding` [size policies](http://doc.qt.io/qt-5/qsizepolicy.html#Policy-enum) are changing it, but not in a good way:

![screen shot 2018-06-10 at 00 25 55](https://user-images.githubusercontent.com/5748627/41196281-08c47130-6c45-11e8-8e03-9d053c202c16.png)

![screen shot 2018-06-10 at 00 26 00](https://user-images.githubusercontent.com/5748627/41196282-0a1fe60e-6c45-11e8-81df-bf6e2e91b402.png)

I think it's a Qt bug.

**Comment by justinclift** [comment]
Well... showing too much space seems better than cutting off the text. :smile:

**Comment by revolter** [comment]
True, but it's quite ugly :(
